apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/QueryOptions.cs

```diff
@@ -131,6 +131,20 @@ public sealed class QueryOptions : JobCreationOptions
         /// </summary>
         public TimePartitioning TimePartitioning { get; set; }
 
+        /// <summary>
+        /// Connection properties which can modify the query job behavior.
+        /// Currently, only the 'session_id' connection property is supported, and is
+        /// used to resolve _SESSION appearing as the dataset id.
+        /// </summary>
+        public IList<ConnectionProperty> ConnectionProperties { get; set; }
+
+        /// <summary>
+        /// If true, creates a new session, where session id will be a server generated random id.
```

Comment:
Nothing needs to be passed in the ConnectionProperties for this case. Because at the moment only session_id is supported in ConnectionProperties and setting this to true will generates it.